etc/docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.3"
+
+services:
+
+  enterprise-gateway:
+    image: elyra/enterprise-gateway:${VERSION:-2.0.0rc1}
```

Comment:
The "convention" we're adopting is to use a `:dev` tag in the various container yaml files, then add entries to `release.sh` that [replace dev tags with release-specific values](https://github.com/jupyter/enterprise_gateway/blob/master/release.sh#L192) at the time a release is produced.

I'm noticing this same issue exists in the docker-based shell script files, so if you want to address those at the same time, that would be great!  This might be moot should we decide that the compose scripts completely replace the shell scripts.

Reply:
That's great I'll fix up the branch to point to `dev` by default.

The shell scripts _should_ be able to be deprecated as the single compose file as it will work in both compose and swarm modes.